cmd/yurthub/app/options/options.go

```diff
@@ -130,6 +148,9 @@ func (o *YurtHubOptions) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&o.HubAgentDummyIfIP, "dummy-if-ip", o.HubAgentDummyIfIP, "the ip address of dummy interface that used for container connect hub agent(exclusive ips: 169.254.31.0/24, 169.254.1.1/32)")
 	fs.StringVar(&o.HubAgentDummyIfName, "dummy-if-name", o.HubAgentDummyIfName, "the name of dummy interface that is used for hub agent")
 	fs.StringVar(&o.DiskCachePath, "disk-cache-path", o.DiskCachePath, "the path for kubernetes to storage metadata")
+	fs.StringVar(&o.CAFile, "ca-file", "", "the CA for yurthub to verify client")
+	fs.StringVar(&o.CertFile, "tls-cert-file", "", "the tls cert of yurthub")
+	fs.StringVar(&o.KeyFile, "tls-private-key-file", "", "the tls key of yurthub")
```

Comment:
It's not convenient for user to setup server certifcates. I think Yurthub shoud generate server certificates by itself.

Reply:
The latest code has been commited. Yurthub generated server certificates by itself.